src/main/java/spoon/reflect/CtModelImpl.java

```diff
@@ -135,12 +139,22 @@ public CtPackage getRootPackage() {
 		return Collections.unmodifiableCollection(getElements(new TypeFilter<>(CtPackage.class)));
 	}
 
+	@Override
+	public CtModule getUnnamedModule() {
+		return this.unnamedModule;
+	}
+
+	@Override
+	public Collection<CtModule> getAllModules() {
+		return this.unnamedModule.getFactory().Module().getAllModules();
```

Comment:
Shorter `return this.unnamedModule.getAllModules();` should return the same result now.

Reply:
Indeed.